internalsdk/pro/pro_test.go

```diff
@@ -1,37 +1,91 @@
 package pro
 
-import (
-	"context"
-	"net/http"
-	"testing"
+// Todo
+// @Paul we might need to find way to use mock here rather calling actual api
```

Comment:
Yeah we should definitely switch to using something like https://github.com/vektra/mockery

Reply:
@atavism What do you think about something like much simple and Morden :- https://github.com/h2non/gock

Reply:
@jigar-f That one looks even better. Nice find!

Reply:
We use [this](https://github.com/uber-go/mock) for mocking some things in lantern-cloud, fwiw

Reply:
...although it seems like we have at least one other file that specifies it was created by Mockery, soooo yay for that